advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py

```diff
@@ -90,36 +94,35 @@ def set_description(self, description):
         self.repaint()
 
 
-class StepSelector(QtGui.QWidget):
+class StepSelector(QtWidgets.QWidget):
 
     """
     The user can select the step to calibrate through this widget.
     """
 
     def __init__(self, parent, calibrer):
-        QtGui.QWidget.__init__(self, parent=parent)
+        QtWidgets.QWidget.__init__(self, parent=parent)
 
         self.current_step_name = None
         self.current_row = 0
         self.steps = {}
         self.steps_description = {}
 
-        self.frame = QtGui.QFrame()
+        self.frame = QtWidgets.QFrame()
         self.calibrer = calibrer
-        self.layout = QtGui.QVBoxLayout()
+        self.layout = QtWidgets.QVBoxLayout()
         self.layout.setSpacing(5)
 
-        self.title = QtGui.QLabel()
+        self.title = QtWidgets.QLabel()
         self.title.setText("Calibration Steps - 2 substeps by steps")
 
         self.step_describer = StepDescriber(self)
 
-        self.list = QtGui.QListWidget()
+        self.list = QtWidgets.QListWidget()
         first_item = self.refresh_list()
-        self.connect(self.list, QtCore.SIGNAL(
-            'itemClicked(QListWidgetItem*)'), self.step_choosed)
-        self.list.setViewMode(QtGui.QListView.ListMode)
-        self.list.setResizeMode(QtGui.QListView.Adjust)
+        self.list.itemClicked['QListWidgetItem*'].connect(self.step_choosed)
```

Comment:
Sorry, the error is actually in a part you did not change, but affected by qt5 changes as well
https://github.com/shadow-robot/sr-visualization/blob/4dbf89f49bd2c070c2113c7194cccba433a62bd1/advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py#L129

this method does not exist anymore see https://doc.qt.io/qt-5/qlistwidget.html

I don't know why you did not get the error, but this is not valid pyqt5 so should be fixed.